scripts/bot-build.slurm

```diff
@@ -32,6 +35,38 @@ if [ -f ${EXPORT_VARS_SCRIPT} ]; then
 else
     echo "could not find ${EXPORT_VARS_SCRIPT} script in '${PWD}', skipping" >&2
 fi
+
+# First, read if there is a local_tmp defined in the site_config section of cfg/job.cfg
+JOB_CFG=cfg/job.cfg
+inside_site_config=false
+local_tmp_value=""
+while IFS= read -r line; do
+    # Check if we've reached [site_config]
+    if [[ $line =~ ^\[site_config\]$ ]]; then
+        inside_site_config=true
+        continue
+    fi
+
+    # If another section starts and we haven't found local_tmp, stop searching
+    if [[ $line =~ ^\[.*\]$ && $inside_site_config == true ]]; then
+        break
```

Comment:
I'd suggest to change this line.
```suggestion
        inside_site_config=false
```
This would then also work in case the `job.cfg` would look like
```ini
[site_config]
some_key = some_value
[some_strange_empty_section]
[site_config]
local_tmp = i_have_no_reason_to_think_that_the_bot_would_create_such_a_config_file_but_who_knows_that_for_sure
```